.github/workflows/build.yml

```diff
@@ -148,67 +123,51 @@ jobs:
 
       - name: Building NodeJS dependencies
         if: steps.cache-deps.outputs.cache-hit != 'true'
```

Comment:
Only thing I can see is that we might want to check the cache-hit status of `steps.node-modules.outputs.cache-hit` as well - I know it depends on the same file but if one was invalidated somehow or something... /shrug